doc/source/whatsnew/v2.1.0.rst

```diff
@@ -298,6 +298,7 @@ Deprecations
 - Deprecated parameter ``obj`` in :meth:`GroupBy.get_group` (:issue:`53545`)
 - Deprecated positional indexing on :class:`Series` with :meth:`Series.__getitem__` and :meth:`Series.__setitem__`, in a future version ``ser[item]`` will *always* interpret ``item`` as a label, not a position (:issue:`50617`)
 - Deprecated strings ``T``, ``t``, ``L`` and ``l`` denoting units in :func:`to_timedelta` (:issue:`52536`)
+- Deprecated support for returning a tuple from a callable in ``iloc``-indexing. Place callables inside a tuple if you need to generate row and column indices using functions (:issue:`53533`)
```

Comment:
Can you explain what `Place callables inside a tuple if you need to generate row and column indices using functions ` means.

Reply:
In a single sentence? Probably not. Perhaps I should just link to the indexing documentation instead.

Reply:
Done.